pkgs/games/xivlauncher/default.nix

```diff
@@ -44,6 +44,9 @@ in
         --replace 'exec' 'exec ${steam-run}/bin/steam-run'
     '' + ''
       wrapProgram $out/bin/XIVLauncher.Core --prefix GST_PLUGIN_SYSTEM_PATH_1_0 ":" "$GST_PLUGIN_SYSTEM_PATH_1_0"
+      # the reference to aria2 gets mangled and isn't detectable by nix: https://github.com/NixOS/nixpkgs/issues/220065
```

Comment:
This is a bit of a nit, but could you specifically mention that it's being mangled as UTF-16LE?

Reply:
done!